index.d.ts

```diff
@@ -810,11 +824,11 @@ export interface i18n {
    * The use function is there to load additional plugins to i18next.
    * For available module see the plugins page and don't forget to read the documentation of the plugin.
    *
-   * Accepts a class or object
+   * @param module Accepts a class or object
    */
-  use<T extends Module>(
-    module: T | Newable<T> | ThirdPartyModule[] | Newable<ThirdPartyModule>[],
-  ): i18n;
+  use<T extends Module>(module: T | NewableModule<T>): this;
+
+  use<T extends ThirdPartyModule>(module: T[] | Array<NewableModule<T>>): this;
```

Comment:
I split this into two signatures to improve clarity.

Reply:
However, both https://github.com/i18next/i18next/blob/v19.4.4/src/i18next.js#L203-L204 and https://github.com/i18next/i18next/blob/v19.4.4/src/i18next.js#L226-L228 does not seem to suggest the `.use()` method takes an array (and the doc does not mention it either). Are we sure this is valid?

Reply:
Based on the runtime, the signature with `[]` no longer appears correct, seems we should remove it.